docs/UserGuide.md

```diff
@@ -182,7 +196,7 @@ Format: `list can`
 
 :exclamation: *On the application, be sure to click on the “Candidates” tab to view the list of candidates.*
 
-:bulb: *This command is usually used after a* [sort can](#sorting-all-candidates-find-can) *command.*
+:bulb: *This command is usually used after a* [sort can](#sorting-all-candidates-sort-can) *command.*
```

Comment:
Is this the intended outcome? I will create a PR to make the appropriate changes